modules/express/src/fetchEncryptedPrivKeys.ts

```diff
@@ -0,0 +1,46 @@
+import { writeFile } from 'fs';
+import { BitGo } from 'bitgo';
+
+// TODO: set env to 'test' or 'prod'
```

Comment:
Could you add a doc comment to explain how to run this? I think the user needs to do something like `yarn ts-node <file>`

Reply:
added a doc comment